frontend/src/controller/DataService.ts

```diff
@@ -1,4 +1,9 @@
 
+export const FADER_TYPE_VALUE = 1;
+export const FADER_TYPE_HUE = 2;
+export const FADER_TYPE_KELVIN = 3;
+export const FADER_TYPE_RGB = 4;
+
```

Comment:
Try to avoid duplications (ArtNetService)

Reply:
Kann ich die defines aus dem Java Projekt benutzen?
Wenn ja wäre toll, aber wie get das?

Reply:
Oh sorry hab das übersehen ist ja frontend... Du könntest die Werte nur im backend über ein Endpunkt bereitstellen aber das ist ein bisschen übertrieben.